examples/ma_events_example.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
```

Comment:
The `go-sdk`  approaches the event examples by deploying a contract that has a function that emits events, and then sending a transaction to call that function. The get event examples then look for those specific events. https://github.com/onflow/flow-go-sdk/blob/master/examples/query_events/main.go

Something similar could be done here.